src/components/MenuDemo/MenuDemo.js

```diff
@@ -1,20 +1,33 @@
 import React, { useEffect } from 'react';
-import { Link } from 'react-router-dom';
+import { InjectAppServices } from '../../services/pure-di';
 
-const MenuDemo = () => {
+// TODO: rename it to Menu
+const MenuDemo = ({
+  dependencies: {
+    appConfiguration: { dopplerLegacyUrl: dopplerLegacyBaseUrl },
+  },
+}) => {
+  // Ugly patch: it is required because in some pages we do not have the menu
+  // and we should show them without waiting.
+  document.getElementById('root')?.classList.remove('dp-show-page');
   useEffect(() => {
     window['doppler-menu-mfe-configuration'] = {
       dopplerMenuElementId: 'doppler-menu-mfe',
-      dmmUseDummies: true,
+      useDummies: false,
+      dopplerLegacyUrl: dopplerLegacyBaseUrl,
```

Comment:
Mi error, cuando refactorice el nombre no modifique esta lineal correctamente. Debe quedar asi:

```suggestion
      dopplerLegacyBaseUrl,
```

Reply:
👍